legal-copy/legal-templates.js

```diff
@@ -34,7 +32,7 @@ module.exports.templateEnd = `
           <a href="https://www.mozilla.org" class="mozilla-logo"></a>
           <a data-l10n-id="footerLinkLegal" href="https://www.mozilla.org/about/legal/" class="boilerplate">Legal</a>
 <a data-l10n-id="footerLinkPrivacy" href="/privacy" class="boilerplate">Privacy</a>
-<a data-l10n-id="footerLinkTerms" href="/terms" class="boilerplate">Terms of Use</a>
+<a data-l10n-id="footerLinkTerms" href="/terms" class="boilerplate">Terms</a>
```

Comment:
"Terms of Use" vs "Terms". I'm a bit indifferent to which one, but if we go "Terms of Use", we should change "Privacy" to "Privacy Notice", so I just rolled with it and was consistent with the main site's footer.